model/k8s/metrics.yaml

```diff
@@ -30,6 +30,28 @@ groups:
     instrument: gauge
     unit: "By"
 
+  # k8s.pod.network.* metrics
+  - id: metric.k8s.pod.network.io
+    type: metric
+    metric_name: k8s.pod.network.io
+    stability: experimental
+    brief: "Pod network IO"
```

Comment:
```suggestion
    brief: "Number of bytes transmitted by the pod"?
```

Reply:
It's not only transmitted but also received. That's why I guess we used `IO` in the kubeletstats' docs.

Reply:
Makes sense! Is there something more elaborate we can say than "Pod network IO" ?

Reply:
I made it `Network bytes for the Pod`. Guess it's more descriptive?